src/components/Modal/Modal.mdx

```diff
@@ -0,0 +1,134 @@
+import { ArgTypes, Canvas, Description, Meta, Source } from "@storybook/blocks";
+
+import * as ModalStories from "./Modal.stories";
+import { ModalTrigger, useModal } from "./Modal";
+
+<Meta of={ModalStories} />
+
+# Modal
+
+| Component Version | DS Version |
+| ----------------- | ---------- |
+| Added             | `0.1.0`    |
+| Latest            | `1.5.0`    |
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Accessibility](#accessibility)
+- [ModalTrigger](#modaltrigger)
+- [ModalTrigger Component Props](#modaltrigger-component-props)
+- [useModal](#usemodal)
+- [useModal Component Props](#usemodal-component-props)
+- [Content Window Scrolling](#content-window-scrolling)
+
+## Overview
+
+_Note: This component is going through an accessibility review and is not
+recommended for use until it is updated._
+
+## Accessibility
+
+TBD
+
+## ModalTrigger
+
+<Description of={ModalTrigger} />
+
+<Source
+  code={`
+import { ModalTrigger } from "@nypl/design-system-react-components";
+
+//...
```

Comment:
We've taken this out of other code examples.  For consistency, should we do that here as well?

Reply:
I think the `Modal` components are special enough that it does need this example snippet.